app/templates/services/_edit-status.html

```diff
@@ -0,0 +1,34 @@
+{% if service_data.status != 'disabled' %}
+  <form action="{{ url_for('.update_service_status', service_id=service_data['id'] ) }}" method="POST">
```

Comment:
The `url_for` seems to be broken here - I think it's encoding the "presented" version of the Service ID - e.g. :+1: `<form action="/suppliers/services/%5B%275837%27%2C%20%276279%27%2C%20%279545%27%2C%20%275488%27%5D" method="POST">`
Instead of `/suppliers/services/5837627995455488`